webui/react/src/components/ParallelCoordinates.tsx

```diff
@@ -26,15 +26,17 @@ const ParallelCoordinates: React.FC<Props> = ({
 
   useEffect(() => {
     if (!containerRef.current) return;
-
     chartRef.current = new Hermes(containerRef.current);
-    if (disableInteraction) chartRef.current?.disable();
 
     return () => {
       chartRef.current?.destroy();
       chartRef.current = undefined;
     };
-  }, [dimensions, disableInteraction]);
+  }, []);
+
+  useEffect(() => {
+    if (disableInteraction) chartRef.current?.disable();
+  }, [disableInteraction]);
```

Comment:
this works ok now? Didn't you have issues with this approach originally?

Reply:
I did at first, but after the refactor it shouldn't be a problem because the chart isn't getting destroyed and recreated. I'm not seeing any issue wrt this in testing, but if you see something let me know.